src/DependencyInjection/Compiler/DoctrinePhpcrOdmDriverPass.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\ContentTypeBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Reference;
+use Doctrine\Common\Persistence\Mapping\Driver\MappingDriverChain;
+
+class DoctrinePhpcrOdmDriverPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->has('doctrine_phpcr.odm.default_metadata_driver')) {
+            return;
+        }
+
+        $driverDef = $container->getDefinition('doctrine_phpcr.odm.default_metadata_driver');
+        $class = $container->getParameter(str_replace('%', '', $driverDef->getClass()));
+
+        if ($class !== MappingDriverChain::class) {
+            throw new \RuntimeException(sprintf(
+                'PHPCR-ODM is not using the chain metadata driver, got "%s"',
+                $driverDef->getClass()
+            ));
+        }
+
+        $driverDef->addMethodCall('addDriver', [
+            new Reference('cmf_content_type.doctrine.phpcr_odm.field_driver'),
```

Comment:
i don't know. did you check the phpcr-odm bundle for service tags or other things we might use? if not it looks not that bad, only relying on implicit things.

ah hm, default in the service name: does that mean we have one of those metadata driver services per object manager?